utils/frame/benchmarking-cli/src/machine/polkadot_reference_hardware.json

```diff
@@ -0,0 +1,32 @@
+[
```

Comment:
shouldn't this file be in Polkadot repo?

Reply:
We can probably move it there later, right now it is easier to keep here since I would otherwise also need to copy it into the cumulus collator and parachain template.

Reply:
Renamed to `SUBSTRATE_REFERENCE_HARDWARE`.